src/main/java/com/dev/cinema/Main.java

```diff
@@ -54,28 +58,25 @@ public static void main(String[] args) throws AuthenticationException,
         movieSession.setShowTime(date);
         movieSessionService.add(movieSession);
 
-        movieSessionService.findAvailableSessions(1L, LocalDate.now()).forEach(System.out::println);
-
-        System.out.println("AUTH:");
+        movieSessionService.findAvailableSessions(1L, LocalDate.now()).forEach(logger::info);
 
+        logger.info("Starting 'Authentication Service' test . . .");
         AuthenticationService authenticationService =
                 (AuthenticationService) injector.getInstance(AuthenticationService.class);
         User bob = authenticationService.register("bob@gmail.com", "qwerty123456");
-        System.out.println(authenticationService.login("bob@gmail.com", "qwerty123456"));
-
-        System.out.println("CART");
+        logger.info(authenticationService.login("bob@gmail.com", "qwerty123456"));
```

Comment:
Better to wrap this in try-catch and add log.warn in the catch block

Reply:
I agree